core-web/libs/dotcms-models/src/lib/dot-experiments.model.ts

```diff
@@ -45,9 +45,9 @@ export interface GoalCondition {
     value: string;
 }
 
-interface RangeOfDateAndTime {
-    startDate: Date;
-    endDate: Date;
+export interface RangeOfDateAndTime {
+    startDate: number;
+    endDate: number;
```

Comment:
We sure about this? I mean having it as a date allows more control no?

Reply:
the endpoint currently return the date in miliseconds, that will be defined in other task related to the edit scheduling.